src/axom/inlet/Container.cpp

```diff
@@ -1070,14 +1070,22 @@ bool Container::verify() const
 
   // If this container was required, make sure something was defined in it
   bool verified =
-    verifyRequired(*m_sidreGroup, this_container_defined, "Container");
+    verifyRequired(*m_sidreGroup, this_container_defined, "Container", errors);
 
   // Verify this Container if a lambda was configured
   if(this_container_defined && m_verifier && !m_verifier(*this))
   {
     verified = false;
-    SLIC_WARNING(
-      fmt::format("[Inlet] Container failed verification: {0}", m_name));
+    const std::string msg =
```

Comment:
Suggestion: Should this new functionality be wrapped in a macro  `INLET_WARNING(error, msg)`  which logs to the vector if `error` is not null and to slic otherwise?

This would potentially clean up the code and allow for easier changes later?